contrib/debian/changelog

```diff
@@ -1,5 +1,5 @@
-horizen (2.0.14) stable; urgency=medium
+horizen (2.0.15) stable; urgency=medium
 
-  * 2.0.14 release.
+  * 2.0.15 release.
 
- -- Zen Blockchain Foundation <info@horizen.global>  Fri, 15 Jun 2018 04:00:00 +0200
+ -- Zen Blockchain Foundation <info@horizen.global>  Thu, 27 Sep 2018 23:00:00 +0200
```

Comment:
It's important to not that we'll have to change this date if we adjust the intended release date.